src/test/org/codehaus/groovy/transform/TupleConstructorTransformTest.groovy

```diff
@@ -131,10 +131,37 @@ class TupleConstructorTransformTest extends GroovyShellTestCase {
 
             assert new Cat("Felix").toString() == 'Cat(name:Felix)'
             assert new Cat("Felix", 42).toString() == 'Cat(name:Felix, id:42)'
-            //assert new Cat("Felix", 42, 3.5d).toString() == 'Cat(age:3.5, name:Felix, id:42)'
+            assert new Cat("Felix", 42, 3.5d).toString() == 'Cat(age:3.5, name:Felix, id:42)'
             assert new Cat(3.5d).toString() == 'Cat(age:3.5)'
             assert new Cat().toString() == 'Cat()'
         '''
     }
 
+    void testMultipleUsages_groovy7672() {
+        assertScript '''
+            import groovy.transform.*
+            import java.awt.Color
+
+            class Named {
+                String name
+            }
+
+            @ToString(includeSuperProperties=true, ignoreNulls=true, includeNames=true, includeFields=true)
+            @TupleConstructor(force=true, defaults=false)
+            @TupleConstructor(force=true, defaults=false, includeFields=true)
```

Comment:
I think it would be helpful to include a multiple usage example like this in the documentation.